src/pages/components/toggle/style.mdx

```diff
@@ -107,7 +127,7 @@ phrase and any proper nouns capitalized, and no more than three words.
 </div>
 
```

Comment:
@alina-jacob, should we add "Label" to this image (first image under 'Small toggle') to show its specs, similar to what you've done in the other image (second image under 'Color')?

(first image)
![Uploading Screenshot 2024-08-26 at 9.50.13 PM.png…]()

(second image)
![image](https://github.com/user-attachments/assets/e9807000-bada-4f39-bd35-ca3663a6d506)

Reply:
Yes, let me create and add an image for this one, thanks!